### ISSUE TITLE
Errors with linebreak-style: ["error", "windows"]

### ISSUE THREAD

**Issue by orlov-vo** [sveltejs/eslint-plugin-svelte3#33]
Hello!

I have errors in Svelte files when I have CRLF symbols on the end of lines and enabled rule `linebreak-style` with `windows` style:

```
  28:0   error    Expected linebreaks to be 'CRLF' but found 'LF'  linebreak-style
  36:18  error    Expected linebreaks to be 'CRLF' but found 'LF'  linebreak-style
  36:29  error    Expected linebreaks to be 'CRLF' but found 'LF'  linebreak-style
```

I think a cause of the problem is in this lines:

https://github.com/sveltejs/eslint-plugin-svelte3/blob/db8aa82022addc212cc8132d1d3fdf0243102997/index.js#L131
https://github.com/sveltejs/eslint-plugin-svelte3/blob/db8aa82022addc212cc8132d1d3fdf0243102997/index.js#L142

**Comment by Conduitry** [comment]
This _should_ be fixed in 2.7.2. Can you try it out please?

**Comment by orlov-vo** [comment]
Yes, it is work for me now!
